css/style.css

```diff
@@ -0,0 +1,52 @@
+#welcomeDiv {
+    width: 100%;
```

Comment:
- [ ] Please, make your code as DRY as it can be. You have done a fantastic job already of keeping it DRY but it can get better with a few changes 😁! In lines 2, 13, and 28 of your CSS file, you are repeating the width property with the same value. You could make a class and reuse it instead of having multiple instances.

Reply:
- [ ] Kindly make your comments more like suggestions instead of orders. This way the developer will be more open to understanding and making the necessary changes, in addition, your reviews will look more professional ✅.

Reply:
- [ ] Kindly make your comments more like suggestions instead of orders. This way the developer will be more open to understanding and making the necessary changes, in addition, your reviews will look more professional ✅.